open_humans/templates/socialaccount/signup.html

```diff
@@ -0,0 +1,154 @@
+{% extends 'panel.html' %}
+
+{% load i18n %}
+{% load account socialaccount %}
+{% load utilities %}
+
+
+{% block head_title %}Create an Open Humans account{% endblock %}
+
+{% block panel_content %}
+
+<div class="well">
+  <p>Members must have a <strong>public username and public
+      name</strong>. You can use any name if it meets our
+    <a href="{% url 'community_guidelines' %}#naming">Naming Guidelines</a>.
+  </p>
+
+  <p>
+    Imported data will be private by default. An account should be yours, or
+    someone for whom you are a legal guardian.
+  </p>
+
+  <p>
+    This site is intended only for United States residents, but we
+    strive to meet EU GDPR standards. <a href="{% url 'gdpr' %}">Read more
+      here.</a>
+  </p>
+</div>
+
+
+<form class="signup" id="signup_form" method="post"
```

Comment:
@jhdulaney the class and id here seem to not be used anywhere else – is that correct?

Reply:
That template is a modification of the one that ships with allauth; that particular class and id comes from there.